arangod/Cluster/ClusterIndexMethods.cpp

```diff
@@ -47,8 +50,26 @@
 #include <memory>
 
 using namespace arangodb;
+namespace paths = arangodb::cluster::paths::aliases;
 
 namespace {
+
+inline auto pathCollectionInTarget(std::string_view databaseName,
+                                   std::string_view cid) {
+  return paths::target()
+      ->collections()
+      ->database(std::string{databaseName})
+      ->collection(std::string{cid});
+}
+
+inline auto pathCollectionIndexesInPlan(std::string_view databaseName,
```

Comment:
This should be renamed to `pathCollectionInPlan`. It gives the impression that we're going to get the indexes.